.github/workflows/ci.yml

```diff
@@ -52,6 +52,30 @@ jobs:
           find examples_formatted -regex '.*\.\(cpp\|hpp\)' -exec clang-format-9 -style=file -i {} \;
           diff -r examples examples_formatted
 
+  documetation:
+    needs:
+      - codespell
```

Comment:
Consider to replace with:
```yaml
  needs: [codespell]
```
As for me, a list with one item written in one line looks better